app/controllers/users.js

```diff
@@ -1,21 +1,26 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
+let mongoose = require('mongoose'),
```

Comment:
Hey, @clintfidel I don't think you're reassigning this `mongoose` variable. 
Only use `let` when you intend to reassign that variable otherwise use `const`